tests/e2e/specs/activate-and-setup/basic-setup.test.ts

```diff
@@ -34,14 +33,16 @@ describe( 'Store owner can login and make sure WooCommerce is activated', () =>
 } );
 
 describe( 'Store owner can finish initial store setup', () => {
-	it( 'can enable tax rates and calculations', async () => {
-		const wcSettings = new WcSettings( page );
+	const wcSettings = new WcSettings( page );
```

Comment:
Did creating it here work? I thought I tried this before and `page` was undefined, although this would definitely be preferred, no fun creating it in each individual test.

Reply:
@louwie17 Yeah I think this worked fine, I haven't seen any issues.